packages/core/src/components/DataTable.story.tsx

```diff
@@ -126,35 +126,26 @@ export class SearchDemo extends React.Component {
     const { data, search } = this.state;
     // eslint-disable-next-line unicorn/no-fn-reference-in-iterator
     const filteredData = this.filter(search);
-    const button = (
-      <Button inline onClick={this.handleNewData}>
-        New data
-      </Button>
-    );
+    const button = <Button onClick={this.handleNewData}>New Data</Button>;
 
     return (
-      <>
+      <div style={{ height: 500, display: 'flex', flexDirection: 'column' }}>
         <Spacing bottom={2}>
           <Row before={button}>
-            <Input
-              hideLabel
-              inline
-              label="Edit row"
-              name=""
-              value={search}
-              onChange={this.handleChange}
-            />
+            <Input hideLabel label="Edit row" name="" value={search} onChange={this.handleChange} />
           </Row>
         </Spacing>
-
-        <DataTable
-          keys={['number', 'zero']}
-          data={data}
-          filterData={filteredData}
-          selectable
-          expandable
-        />
-      </>
+        <div style={{ flexGrow: 1 }}>
+          <DataTable
+            tableHeaderLabel="My Great Table"
+            data={data}
+            filterData={filteredData}
+            autoHeight
+            expandable
+            columnHeaderHeight="micro"
```

Comment:
small nit: move boolean props to the top to be consistent

Reply:
there's a lint rule for this isn't there?

Reply:
Yeah. Enabled in Nimbus but probably not here.